### PR TITLE
add berberistestnetwork

### PR DESCRIPTION
**Pull request by crypto-niac**
This PR adds Hedges test network called berberis-1

### CODE REVIEW

File: testnets/berberistestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "assets": [
+    {
+      "description": "The native staking token of the Hedge blockchain.",
+      "denom_units": [
+        {
+          "denom": "hedge",
+          "exponent": 0
+        },
+        {
+          "denom": "uhedge",
```
Comment (JeremyParish69) [inline testnets/berberistestnet/assetlist.json:13]: the denoms are backwards here. 'uhedge' should be the first denom with exponent 0, and the 'hedge' after with exponent 6

File: testnets/berberistestnet/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+          "exponent": 6
+        }
+      ],
+      "base": "hedge",
+      "name": "Berberis HEDGE",
+      "display": "hedge",
+      "symbol": "HEDGE"
+    }
```
Comment (JeremyParish69) [inline testnets/berberistestnet/assetlist.json:21]: Please define an image HEDGE

File: testnets/berberistestnet/chain.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      "url": "https://berberis.hedgescan.io",
+      "tx_page": "https://berberis.hedgescan.io/txs/${txHash}"
+    }
+  ],
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/berberistestnet/images/BerberisBrandmark.png",
+      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/berberistestnet/images/BerberisBrandmark.svg"
+    }
+  ]
```
Comment (JeremyParish69) [inline testnets/berberistestnet/chain.json:64]: Please defined all chain images under 'images' here, or else remove all the images that won't be used.

File: testnets/berberistestnet/assetlist.json (hunk 1 above)
Comment (crypto-niac) [reply testnets/berberistestnet/assetlist.json]: Given that my base denom is defined as `hedge`, I believe it is okay to assign an exponent of 0 to `hedge` and an exponent of 6 to `uhedge`.

File: testnets/berberistestnet/assetlist.json (hunk 2 above)
Comment (crypto-niac) [reply testnets/berberistestnet/assetlist.json]: Added

File: testnets/berberistestnet/chain.json (hunk 3 above)
Comment (crypto-niac) [reply testnets/berberistestnet/chain.json]: Removed unused images

File: testnets/hedgetestnet/chain.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  "website": "https://hedgeblock.io",
+  "bech32_prefix": "hedge",
+  "daemon_name": "hedged",
+  "node_home": "$HOME/.hedge",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "uhedge",
```
Comment (JeremyParish69) [inline testnets/hedgetestnet/chain.json:17]: Since the base is 'hedge', please change this to 'hedge'

File: testnets/hedgetestnet/assetlist.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "uhedge",
+          "exponent": 6
+        }
+      ],
+      "base": "hedge",
+      "name": "Berberis HEDGE",
+      "display": "hedge",
```
Comment (JeremyParish69) [inline testnets/hedgetestnet/assetlist.json:19]: I assume the token shouldn't be displayed in the base form. The display denom should match whichever you use for the exponent 6.

File: testnets/berberistestnet/assetlist.json (hunk 1 above)
Comment (crypto-niac) [reply testnets/berberistestnet/assetlist.json]: I understood where I might have gone wrong in my denom_metadata. I will get on to that and update in my genesis. If I do that the following changes makes no sense and I need to roll back to how it was earlier except the denom_metadata. Will update in our chain as well.

Comment (crypto-niac) [reply testnets/berberistestnet/assetlist.json]: This should resolve our confusions. https://github.com/cosmos/chain-registry/pull/3806/commits/2d1ac1c73951bb89aa033fb4a57e5cf4dbb4300a

**Pull request merged**